### PR TITLE
Centralise padding for new ui pages

### PR DESCRIPTION
**Pull request by PenghaiZhang**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] tests are included
- [x] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change

1. Centralise CSS padding for the main content area of New UI so that each page does not need to duplicate styles. 

2. Refactor Login notice editor and Theme page so that they have consistent padding as other pages.

Screenshot from 2020-06-24 16-16-11

Screenshot from 2020-06-24 15-52-03

Screenshot from 2020-06-24 15-52-19

Screenshot from 2020-06-24 15-52-37

Screenshot from 2020-06-24 15-52-52

Screenshot from 2020-06-24 15-53-11

Screenshot from 2020-06-24 15-53-25

### CODE REVIEW

File: Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/ContentIndexSettings.tsx [hunk 1]
```diff
@@ -131,82 +112,88 @@ function ContentIndexSettings({ updateTemplate }: TemplateUpdateProps) {
...
+                <Slider
+                  disabled={showError}
+                  marks={boostVals}
+                  min={0}
+                  max={7}
+                  getAriaValueText={getAriaLabel}
+                  aria-label={contentIndexSettingsStrings.titleBoostingTitle}
+                  value={searchSettings.titleBoost}
+                  onChange={(event, value) =>
+                    handleSliderChange(value as number, "titleBoost")
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/ContentIndexSettings.tsx:153]: would it make sense to do a runtime check like:
```ts
if (typeof value !=== 'number') throw new TypeError('slider value must be a number')
```
?

File: Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/ContentIndexSettings.tsx [hunk 2]
```diff
@@ -131,82 +112,88 @@ function ContentIndexSettings({ updateTemplate }: TemplateUpdateProps) {
...
+                <Slider
+                  disabled={showError}
+                  marks={boostVals}
+                  min={0}
+                  max={7}
+                  getAriaValueText={getAriaLabel}
+                  aria-label={contentIndexSettingsStrings.metaBoostingTitle}
+                  value={searchSettings.descriptionBoost}
+                  onChange={(event, value) =>
+                    handleSliderChange(value as number, "descriptionBoost")
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/ContentIndexSettings.tsx:171]: would it make sense to do a runtime check like:
```ts
if (typeof value !=== 'number') throw new TypeError('slider value must be a number')
```
?

File: Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx [hunk 3]
```diff
@@ -130,138 +111,142 @@ function SearchPageSettings({ updateTemplate }: TemplateUpdateProps) {
...
+                <SettingsToggleSwitch
+                  value={searchSettings.searchingShowNonLiveCheckbox}
+                  setValue={(value) =>
+                    setSearchSettings({
+                      ...searchSettings,
+                      searchingShowNonLiveCheckbox: value,
+                    })
+                  }
+                  disabled={showError}
+                  id={"_showNonLiveCheckbox"}
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx:150]:
```suggestion
                  id="_showNonLiveCheckbox"
```

File: Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx [hunk 4]
```diff
@@ -130,138 +111,142 @@ function SearchPageSettings({ updateTemplate }: TemplateUpdateProps) {
...
+                <SettingsToggleSwitch
+                  value={searchSettings.authenticateFeedsByDefault}
+                  setValue={(value) =>
+                    setSearchSettings({
+                      ...searchSettings,
+                      authenticateFeedsByDefault: value,
+                    })
+                  }
+                  disabled={showError}
+                  id={"_authenticateByDefault"}
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx:169]:
```suggestion
                  id="_authenticateByDefault"
```

File: Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx [hunk 5]
```diff
@@ -130,138 +111,142 @@ function SearchPageSettings({ updateTemplate }: TemplateUpdateProps) {
...
+              primaryText={searchPageSettingsStrings.cloudSearching}
+              secondaryText={searchPageSettingsStrings.cloudSearchingLabel}
+              control={
+                <SettingsToggleSwitch
+                  value={cloudSettings.disabled}
+                  setValue={(value) =>
+                    setCloudSettings({ ...cloudSettings, disabled: value })
+                  }
+                  disabled={showError}
+                  id={"cs_dc"}
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx:184]:
```suggestion
                  id="cs_dc"
```

File: Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx [hunk 6]
```diff
@@ -130,138 +111,142 @@ function SearchPageSettings({ updateTemplate }: TemplateUpdateProps) {
...
+                <SettingsToggleSwitch
+                  value={searchSettings.searchingDisableGallery}
+                  setValue={(value) =>
+                    setSearchSettings({
+                      ...searchSettings,
+                      searchingDisableGallery: value,
+                    })
+                  }
+                  disabled={showError}
+                  id={"_disableGallery"}
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx:209]:
```suggestion
                  id="_disableGallery"
```

File: Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx [hunk 7]
```diff
@@ -130,138 +111,142 @@ function SearchPageSettings({ updateTemplate }: TemplateUpdateProps) {
...
+                <SettingsToggleSwitch
+                  value={searchSettings.searchingDisableVideos}
+                  setValue={(value) =>
+                    setSearchSettings({
+                      ...searchSettings,
+                      searchingDisableVideos: value,
+                    })
+                  }
+                  disabled={showError}
+                  id={"_disableVideos"}
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx:227]:
```suggestion
                  id="_disableVideos"
```

File: Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx [hunk 8]
```diff
@@ -130,138 +111,142 @@ function SearchPageSettings({ updateTemplate }: TemplateUpdateProps) {
...
+                <SettingsToggleSwitch
+                  value={searchSettings.fileCountDisabled}
+                  setValue={(value) =>
+                    setSearchSettings({
+                      ...searchSettings,
+                      fileCountDisabled: value,
+                    })
+                  }
+                  disabled={showError}
+                  id={"_disableFileCount"}
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx:244]:
```suggestion
                  id="_disableFileCount"
```

File: Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/searchfilter/SearchFilterSettingsPage.tsx [hunk 9]
```diff
@@ -284,77 +279,81 @@ const SearchFilterPage = ({ updateTemplate }: TemplateUpdateProps) => {
...
+              control={
+                <SettingsToggleSwitch
+                  value={searchSettings.searchingDisableOwnerFilter}
+                  setValue={(disabled) => {
+                    setSearchSettings({
+                      ...searchSettings,
+                      searchingDisableOwnerFilter: disabled,
+                    });
+                  }}
+                  id={"disable_owner_filter_toggle"}
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/searchfilter/SearchFilterSettingsPage.tsx:297]:
```suggestion
                  id="disable_owner_filter_toggle"
```

File: Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/searchfilter/SearchFilterSettingsPage.tsx [hunk 10]
```diff
@@ -284,77 +279,81 @@ const SearchFilterPage = ({ updateTemplate }: TemplateUpdateProps) => {
...
+              control={
+                <SettingsToggleSwitch
+                  value={searchSettings.searchingDisableDateModifiedFilter}
+                  setValue={(disabled) => {
+                    setSearchSettings({
+                      ...searchSettings,
+                      searchingDisableDateModifiedFilter: disabled,
+                    });
+                  }}
+                  id={"disable_date_modified_filter_toggle"}
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/searchfilter/SearchFilterSettingsPage.tsx:312]:
```suggestion
                  id="disable_date_modified_filter_toggle"
```

**Comment by edalex-ian** [comment]
Merge on @PenghaiZhang 

The tests all passed, that was just saving the coverage report. And what are the chances I also just raise a PR to hopefully help that in the future: #1937

**Pull request merged**